README.md

```diff
@@ -110,6 +115,78 @@ You can verify your smart contract on Etherscan by running:
 yarn verify --network network_name
 ```
 
+## Hook Example
+
+- useScaffoldContractRead
+- useScaffoldContractWrite
+- useScaffoldEventSubscriber
+- useScaffoldEventHistory
+- useDeployedContractInfo
+
+  **useScaffoldContractRead**:
+  Use this hook to read a value from your deployed contracts.
+
+  ```ts
+  const { data: totalCounter } = useScaffoldContractRead({
+    contractName: "YourContract",
+    functionName: "getGreeting",
+    // if it does not accept any args it should be args: []
```

Comment:
The args param can bee omitted entirely when no args needed for the function call, no need to pass empty array.

Reply:
ok,